src/graphics/program-lib/chunks/clearCoatGloss.frag

```diff
@@ -21,5 +21,9 @@ void getClearCoatGlossiness() {
     ccGlossiness *= saturate(vVertexColor.$VC);
     #endif
 
+    #if GLTF_MAT_GENERATED
+    dGlossiness = 1.0 - dGlossiness;
```

Comment:
is this a better solution than setting roughness like this on the material directly at load time in the glb parser?

Reply:
Up until the invert, glossiness is actually really roughness - so we could indeed invert it earlier - but we would have to invert all the components that can make up the roughness including the roughness factor - which is easy and roughness map texture channel which is trickier. Alternatively we could just invert the texture sample - and convert roughness factor?

Reply:
Hmm looks like I made a mistake in this PR - not dealing with specular-glossiness extension correctly - will fix this

Reply:
spec-gloss fixed and only roughness texture inverted in shader chunk